src/Brand/ReaderBrand.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Marcosh\LamPHPda\Brand;
+
+use Marcosh\LamPHPda\Reader;
+
+/**
+ * @implements Brand<Reader>
+ */
```

Comment:
even though the class is empty, I'd use `@psalm-immutable` anyway

Reply:
Fixed.